dashboard/urls.py

```diff
@@ -40,4 +40,5 @@
         name='ping_json'),
     url(r'^healthcheck.json$', HealthcheckView.as_view(),
         name='healthcheck_json'),
+    url(r'^admin/login/$', 'django.contrib.auth.views.login'),
```

Comment:
i see why you need this line now, but still cannot figure out why it's the case. let's leave with it.